test/h-group/level-7.js

```diff
@@ -47,7 +47,7 @@ describe('scream discard chop moves', () => {
 		});
 
 		takeTurn(game, 'Alice discards r1 (slot 5)');	// End early game
-		takeTurn(game, 'Bob clues green to Cathy');
+		takeTurn(game, 'Bob clues 1 to Cathy');
```

Comment:
This was not being treated as a SDCM because of the possibility of Cathy seeing a G1 in our hand. I think it'd be easy to force it to assume that it's playable in this case if we think it should.

Reply:
I think this should SDCM even if Cathy sees a g1 in our hand, Bob just gave a clue that makes Cathy's intentions ambiguous. At least, that's how I would play.

Reply:
Isn't it bad to have accidental SDCM's though everytime there's an ambiguous play clue given? My thinking was that this could be very surprising.

Reply:
I suppose the common case is that you don't have the matching card and it'd be better to support SDCM in this case even though it'd be a bit of out of sync during the symmetric connection case. I'll update the code for this.